docsrc/antora-extension/index.js

```diff
@@ -0,0 +1,172 @@
+'use strict'
+
+// Helpers for Handlebars (but also for general use)
+const helpers = {
+
+  toUpper: (s) => {
+    return s.toUpperCase()
+  },
+
+  upperFirst: (s) => {
+    return s.charAt(0).toUpperCase() + s.slice(1)
+  },
+
+  // Convert "foo_bar" to "foo bar"
+  toWords: (s) => {
+    return s.replace(/_/g, " ")
+  },
+
+  // Convert "foo_bar" to "Foo bar"
+  toTitle: (s) => {
+    return helpers.upperFirst(helpers.toWords(s))
+  },
+
+  // https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore#_groupby
+  groupBy: (xs, f) => {
+    return xs.reduce((r, v, i, a, k = f(v)) => ((r[k] || (r[k] = [])).push(v), r), {});
+  },
+}
+
+module.exports.register = function() {
+  this.on('contentAggregated', ({ contentAggregate }) => {
+
+    // Import modules for yaml and handlebars
+    const yaml = this.require('js-yaml')
+    const Handlebars = this.require('handlebars')
+
+    // Register the helpers so they're available in templates
+    Object.keys(helpers).forEach((k) => {
+      Handlebars.registerHelper(k, helpers[k])
+    })
+
+    // ROOT module
+    const content = contentAggregate[0]
+
+    // Ignore everything except for the annotation-data.json file
+    content.files.filter(f => f.src.basename === 'annotations-data.json').forEach(f => {
+
+      // Load the rego annotations data
+      // It's json but yaml.load will happily read it
+      const rawData = yaml.load(f.contents)
```

Comment:
For JSON, you can use the builtin [JSON.parse](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/parse)

Reply:
Decided to use yaml anyway to save a few lines of code.